admin/lib/generators/solidus_admin/install/install_generator.rb

```diff
@@ -24,6 +26,18 @@ def ignore_tailwind_build_files
       def build_tailwind
         rake "solidus_admin:tailwindcss:build"
       end
+
+      def install_lookbook
+        return unless options[:lookbook]
+
+        gem_group :development, :test do
+          gem "lookbook"
+          gem "listen"
+          gem "actioncable"
+        end
+
+        route "mount Lookbook::Engine, at: '/lookbook' if Rails.env.development?"
```

Comment:
@elia can you help me understand why we need `if Rails.env.development?"` if we also have the ENV variable?

Reply:
That is how lookbook suggests to add it to routes, the same goes for the bundler groups the gems will belong to. Also the env variable is about whether we'll install it, not for when it should be active.

I think it's good to stick to lookbook's instructions when we install it to the host app.